src/App.jsx

```diff
@@ -1,5 +1,48 @@
+/** @jsxImportSource @emotion/react */
+import { css } from '@emotion/react';
+import Header from './components/Header';
+import Content from './components/Content';
+import InputBox from './components/InputBox';
+import GlobalStyle from './components/GlobalStyle';
+import { useState } from 'react';
+import messageData from './db/data.json';
+
 function App() {
-  return <div>React SNS</div>;
+  const [messages, setMessages] = useState(messageData.message);
+
+  function addMessage(text) {
+    const newMessage = {
+      id: messages.length + 1,
+      userId: 1,
+      text: text,
+      time: new Date().toLocaleTimeString([], {
+        hour: '2-digit',
+        minute: '2-digit',
+      }),
+    }; // '오전 12:00' 포맷으로 출력
+
+    setMessages((prevMessages) => [...prevMessages, newMessage]); // 기존 메시지에 새로운 메시지 추가
+  }
+
+  return (
+    <div css={wrapperStyle}>
+      <GlobalStyle />
+      <Header />
+      <Content messages={messages} />
+      <InputBox addMessage={addMessage} />
+    </div>
```

Comment:
```suggestion
<>
   <GlobalStyle />
    <div css={wrapperStyle}>
      <Header />
      <Content messages={messages} />
      <InputBox addMessage={addMessage} />
    </div>
</>
```
`GlobalStyle`은 말 그대로 모든 요소에 전역적으로 설정하기 위한 파일이라 가장 바깥에 위치시키는 게 그 의미에 더 맞을 것 같아요!